### ISSUE TITLE
Consolidation of Post-V2 Character/Glyph Addition Suggestions

### ISSUE THREAD

**Issue by kenlunde** [adobe-fonts/source-han-sans#206]
The following table shows the glyph additions that are planned for after the Version 2.000 update, and unless otherwise noted, the additions are from my own notes:

**Character** | **Glyph Name** | **Notes**
--- | --- | ---
U+N/A | _N/A_ | N/A

**Comment by tamcy** [comment]
As the final stroke of the 木 component on the top-right is different, glyphs of the following characters cannot be shared with other regions:

Currently shared with CN:
U+3A52 㩒
U+3BF2 㯲
U+7981 禁
U+6983 榃

Currently shared with JP:
U+50F8 僸

**Comment by kenlunde** [comment]
@KrasnayaPloshchad Negative. The number of required glyphs is too high, and the glyph set is already full. BTW, the feature tag to which you're referring is '[ital](https://docs.microsoft.com/en-us/typography/opentype/spec/features_fj#a-nameital-iditalatag-39ital39)'.

**Comment by Stone-Zeng** [comment]
Similar issues have been raised for several times (#191 and <https://github.com/adobe-fonts/source-han-serif/issues/75>).

Now, the italic shape of Source Serif Pro has been made, while the HK support of Source Han Sans has been added as well. So do you think it's time to support italics? (perhaps I should open a new issue)

**Comment by kenlunde** [comment]
@Stone-Zeng Why do you think it would be a good idea to open a new issue about adding italics? Adding HK support to _Source Han Serif_ will result in an overflowing glyph set that will require a special touch to fit within the 65,535-glyph limit, meaning that adding glyphs for italics is an effective non-starter.

**Comment by Stone-Zeng** [comment]
I don't know whether `ital` feature is necessary. Why not separate roman and italics into different otf/otc files, so that the 65,535-limit is no longer a limit? Anyway, I think many people will be happy if a properly-adjusted italic shape could be directly used altogether with Source Han Sans/Serif.

**Comment by frankrolf** [comment]
I agree with @Stone-Zeng in thinking that a separate Italic OTC would be a nice idea, however I don’t know how common that division is/how frequently the `ital` feature is used. It may well be

One remark:
When you say “the Italic shape of Source Serif Pro has been made”, you need to consider that Source Serif and Source Han Serif are quite different in terms of their setup – Source Han Serif uses four masters for Latin, while Source Serif only uses three.
Also, Source Han Serif includes a big number of additional Western glyphs that don’t ship with Source Serif – which presumably also would need Italics.
In short: it’s not that simple, and creating Source Han Serif Italic would mean a considerable amount of work.

**Comment by Buernia** [comment]
CN, JP and KR glyph of 燊 (U+71CA) should be reverted to pre-V2 glyph.
![shs](https://user-images.githubusercontent.com/11886946/50674852-d5012580-1024-11e9-8706-223b5c549a21.png)

Below is the glyph in 通用规范汉字表：
![default](https://user-images.githubusercontent.com/11886946/50675014-307fe300-1026-11e9-84ea-6502550c39b0.png)

**Comment by tamcy** [comment]
The top-left component of HK's 虩 (U+8669) is different from TW, so the glyph cannot be shared. 

![u8669](https://user-images.githubusercontent.com/959433/53709297-74e6fe00-3e72-11e9-8ea3-ac50be8f4952.png)

**Comment by tamcy** [comment]
(This entry is about *U+961D*. *U+2ED6* and *U+2ECF* are included in the screenshot for reference.)

![uni961D](https://user-images.githubusercontent.com/959433/57177828-c889be00-6e9a-11e9-9f04-d12f8135640d.png)

Currently there's only one glyph for *U+961D*, which is *uni961D-CN*. It points to CID 43066, the same as *U+2ED6*.

However, *U+961D* is also a HKSCS character (P.252 of the reference glyph document), and its shape is centered to the em-box:
<img src="https://user-images.githubusercontent.com/959433/57177943-1521c900-6e9c-11e9-8ae6-5617965389a6.png" width="100">

The decision to center the character in the em box is the same for other regions except for CN ([Unicode code chart](https://www.unicode.org/charts/PDF/Unicode-12.0/U120-4E00.pdf) P.463):
![uni961D-2](https://user-images.githubusercontent.com/959433/57177945-1f43c780-6e9c-11e9-8f95-3c6ce5277e12.png)



So a new glyph would be needed for HK, which can be further shared to all other non CN regions.

**Comment by Buernia** [comment]
CN, JP and KR glyph of 胣 (U+80E3) should be reverted to pre-V2 glyph.
![图片](https://user-images.githubusercontent.com/11886946/69933461-cf001700-1509-11ea-9b68-5ff769340760.png)

**Comment by byyxx128** [comment]
### How about "𫟷" as the simplified Chinese name of the 116th element?
![image](https://user-images.githubusercontent.com/20079289/130553727-c6b48402-8877-4bf2-bb55-c6c8540c76e6.png)
### I did not find the latest progress about it, since it was still available.

**Comment by byyxx128** [comment]
### How about "𫟷" as the simplified Chinese name of the 116th element?
![image](https://user-images.githubusercontent.com/20079289/130553727-c6b48402-8877-4bf2-bb55-c6c8540c76e6.png)
### I did not find the latest progress about it, since it was still unavailable.

**Comment by punchcutter** [comment]
@byyxx128 What's the question? 𫟷 is in Source Han Sans.

**Comment by byyxx128** [comment]
> What's the question? 𫟷 is in Source Han Sans.

@punchcutter I tried different Font Types, `𫟷` is only missing in SubsetOTF.
https://github.com/adobe-fonts/source-han-sans/tree/release/SubsetOTF/CN

In addition, some characters'(e.g. `𬭛𫟼`) style also looks different from other characters' (e.g. `𬬻𬭊𬭳𬭶鿏𬬭鿔鿭𫓧镆𫟷`).
![image](https://user-images.githubusercontent.com/20079289/130597308-ad8ef8fe-6215-40ef-bb99-498003cbd8ca.png)
![image](https://user-images.githubusercontent.com/20079289/130597599-c56d192e-ada9-4783-afd5-fa205832c1bf.png)
Weight of font: Light 👆

Not every weight has the problem.
![image](https://user-images.githubusercontent.com/20079289/130598396-03ef24ad-b5b3-4f4b-a5c3-27c975401a1e.png)
Weight of font: **Heavy** 👆

## You may need to test all weights of the fonts.

**Comment by punchcutter** [comment]
@byyxx128 I see. We can add u2B7F7-CN to the CN Subset and check the balance on those other glyphs.
